src/parser.jl

```diff
@@ -866,6 +866,7 @@ function parse!(r::Union{Request, Response}, parser, bytes, len=length(bytes);
                             @err(HPE_BODY_OVERFLOW)
                         end
                         parser.content_length = t
+                        r.body.expectedlength = Int64(t)
```

Comment:
Underscores help non-English speakers parse out words as the separation may not be intuitive